src/lib/adsb/AdsbConflict.cpp

```diff
@@ -129,9 +131,16 @@ void AdsbConflict::get_traffic_state()
 
 	if (old_conflict && _conflict_detected) {
 		old_conflict_warning_expired = (hrt_elapsed_time(&_traffic_buffer.timestamp[traffic_index]) > CONFLICT_WARNING_TIMEOUT);
-
 	}
 
+	PX4_DEBUG("Timestamp: %lu \n",
+		  _traffic_buffer.timestamp[traffic_index]);
+	PX4_DEBUG("Time now: %lu \n", hrt_absolute_time());
+	PX4_DEBUG("Time since timestamp: %lu \n",
+		  hrt_elapsed_time(&_traffic_buffer.timestamp[traffic_index]));
+	PX4_DEBUG("Old Conflict warning expired: %d \n", old_conflict_warning_expired);
+	PX4_DEBUG("--------------------\n");
+
```

Comment:
Did you mean to add this?

Reply:
Of course, we don't need it. I'm just being lazy so I don't need to add another time if this CI fails again. I can remove.

Reply:
removed